libs/ui/packages/react/src/components/message/Alert/index.tsx

```diff
@@ -91,28 +103,33 @@ export default function Alert({
   containerProps,
 }: AlertProps): JSX.Element {
   const theme = useTheme();
-  const { color, background } = getColors({ theme, type });
+  const { iconColor, background } = getColors({ theme, type });
   const textProps: { variant?: TextVariants; fontWeight?: string } = {
     variant: "paragraph",
     fontWeight: "medium",
   };
```

Comment:
variant should now be `bodyLineHeight` and fontWeight `semiBold`

Reply:
I have changed lineHeight to 21px too.

Reply:
let's check with designers if line height variants should all adopt this rather than making it a particular rule here for this specific case ...

Reply:
so it's back to 170% in the design, you can remove the places where you put 1.5
https://www.figma.com/file/WXCEmRMwW47ELrFonzPTGg?node-id=9509:42966#428079324